### PR TITLE
AITOOL 3059 - Added secure mode section

### PR DESCRIPTION
**Pull request by joseangel-mm**
# [AITOOL-3059](https://lampkicking.atlassian.net/browse/AITOOL-3059)

## What?
Add secure mode section and replace the sfcm diagram with an image.

## Why?
Make sure the npm package page is visible too.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -57,6 +57,37 @@ The usage of this module can be found in `App.js`. You can find more information
...
+
+The flow for the secure mode is the following:
+
+1. Request for a session -> `(GET) https://api.yoti.com/ai/sm/v1/secure-fcm/<version>/token`
+2. Request for the encrypted javascript module -> `(GET) https://api.yoti.com/ai/sm/v1/secure-fcm/<version>/module?s=<token>`
+
+Where `<version>` is the current version of the FCM (It is embedded in the
+module) and the `<token>` is the session token the first request returns.
+
+![Diagram](./docs/sfcm.png)
```
Comment (tonygcs) [inline README.md:89]: Can you use the diagram definition here, please? Github is able to generate the image so we can modify the diagram next time to generate a new image and save it in this repo too in order to use it in the npm doc page.

What do you think?

File: README.md [hunk 2]
```diff
@@ -57,6 +57,37 @@ The usage of this module can be found in `App.js`. You can find more information
 
 Use the secure flag to ensure the image can't be changed once is taken. Have in mind this will make some additional calls to the Yoti servers in order to secure the request.
 
+### Secure mode
```
[the remark this answers is not in the document]
Comment (joseangel-mm) [reply README.md:60]: Yes, it could be better.

File: docs/secure-mode.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+  participant B as Secure Module Server
+  participant C as Integrator back-end
+  participant D as Yoti Anti Spoofing Service
+  A->>B: Request for a session
+  B->>A: New session response
+  A->>B: Request for the secure capture module
+  B->>A: Secure capture module result
+  A->>A: Take picture
+  A->>C: Send capture result
+  C->>D: Propagate request to the Anti Spoofing service
```
Comment (tonygcs) [inline docs/secure-mode.md:42]: Add a screeshot of the diagram in the `doc` folder too, please. You can use it for the FCM npm doc.

Comment (joseangel-mm) [reply docs/secure-mode.md:42]: Sure. Done!

**Review by tonygcs — APPROVED** [review]

File: docs/secure-mode.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+    "token": "<session_token>",
+    "signature": "<result_signature>"
+  }
+}
+```
+
+The flow for the secure mode is the following:
+
+1. Request for a session -> `(GET) https://api.yoti.com/ai/sm/v1/secure-fcm/<version>/token`
+2. Request for the encrypted javascript module -> `(GET) https://api.yoti.com/ai/sm/v1/secure-fcm/<version>/module?s=<token>`
```
Comment (MiguelJS) [inline docs/secure-mode.md:25]: wdyt?
```suggestion
2. Request for the obfuscated javascript module -> `(GET) https://api.yoti.com/ai/sm/v1/secure-fcm/<version>/module?s=<token>`
```

Comment (joseangel-mm) [reply docs/secure-mode.md]: Yes, I think is better. Changed!

**Review by MiguelJS — APPROVED** [review]

**Review by DionisioRD — APPROVED** [review]

**Pull request merged**